### ISSUE TITLE
Steam directory not found

### ISSUE THREAD

**Issue by FridayAnubis** [Exitium-DEV/HD2-Cheat-Launcher#5]
Thanks for your work, but the latest version still shows
"Steam directory not found at the default location: C:\Program Files (x86)\Steam
Steam is not installed or its installation path couldn't be found
Press any key to continue..."
Please fix this.

**Comment by byremo994** [comment]
`Console.WriteLine("IMPORTANT: If your Steam installation directory is not default...\nMake a file 'steam_override.txt' containing the location of your installation directory, and restart the EXE\n\nEXAMPLE: D:\\Some\\Path\\Steam\n");`

Copy paste from the readme. Just make a steam_override.txt file with the installation path.

**Comment by FridayAnubis** [comment]
Ok I fixed this, thanks for the help. By the way, the message reminding users of creating the txt file only shows when there IS a steam_override.txt file, Idk if this is a mistake.

**Comment by Sinnisterly** [comment]
https://prnt.sc/7ryyBM6KSGoN am I just being dumb? having same issue after making steam override.

**Comment by DatGuyMcCaully** [comment]
Yeah, its already detected anyways, but shit seems to not work if you a game library on another drive with the default steam installation.

**Comment by FridayAnubis** [comment]
Well my launcher worked, you need to enter your steamlibrary path where your game was installed, not the steam itself, and put the dll in same folder as the launcher.
But this method doesn't work anymore, I guess we need to find someway else.

**Comment by DatGuyMcCaully** [comment]
D:\Games in there like 4 different times. maybe I had a \ on the end and the fucked it. idk it works now.
